new-lamassu-admin/src/routing/routes.js

```diff
@@ -16,37 +16,15 @@ import Login from 'src/pages/Authentication/Login'
 import Register from 'src/pages/Authentication/Register'
 import Reset2FA from 'src/pages/Authentication/Reset2FA'
 import ResetPassword from 'src/pages/Authentication/ResetPassword'
-import Blacklist from 'src/pages/Blacklist'
-import Cashout from 'src/pages/Cashout'
-import Commissions from 'src/pages/Commissions'
 // import ConfigMigration from 'src/pages/ConfigMigration'
```

Comment:
can you check if we can we remove this since we're editing this file anyway, please?